apps/modernization-ui/src/routes/AppRoutes.tsx

```diff
@@ -92,6 +93,8 @@ export const AppRoutes = () => {
 
                                 {config.features.pageBuilder.enabled && (
                                     <Route path="/page-builder" element={<PageBuilderContextProvider />}>
+                                        {/*  temp add this route for testing*/}
+                                        <Route path="page/:pageId?" element={<EditPage />} />
```

Comment:
Issue: We will not be using the old `EditPage` component. It will be deleted in the near future